DEVELOPMENT.md

```diff
@@ -246,50 +246,81 @@ This section is written primarily for maintainers, so we don't forget how to ini
+
+```console
+$ git checkout -b release-v0.2.1 upstream/release-v0.2.0
+$ git cherry-pick <commit from master>
+....
+```
+
+### Updating the helm chart
+
+Regardless of which branch you created, you'll need to update the helm chart number. After you've cut the branch, make sure to change the versions in the [`Chart.yaml`](https://github.com/gomods/athens/blob/master/charts/athens-proxy/Chart.yaml) file:
+
+- Increment the patch number in the [`version` field](https://github.com/gomods/athens/blob/master/charts/athens-proxy/Chart.yaml#L2)
+- Set the [`appVersion` field](https://github.com/gomods/athens/blob/master/charts/athens-proxy/Chart.yaml#L2) to the semver of the new branch. Do not include the `v` prefix
 
 ## Creating the new release in GitHub
 
 Go to the [create new release page](https://github.com/gomods/athens/releases/new) and draft a new release. See below for what data to put into the fields you see:
 
-- **Tag version** - This should be the same `v0.x.y` number you put into the release branch. Again, make sure this tag starts with `v`
+- **Tag version** - This should be the same `v0.x.y` number you put into the release branch. Make sure this tag starts with `v` and that the tag target is the proper release branch.
 - **Release Title** - Make sure the title is prefixed by the release number including the `v`. If you want to write something creative in the rest of the title, go for it!
-- **Describe this release** - Make sure to write what features this release includes, and any notable bugfixes. Also, thank all the folks who contributed to the release. You can find that information in a link that looks like this: `https://github.com/gomods/athens/compare/master...v0.7.2`. Substitute the `v0.7.2` from the most recent release before the one you are creating
+- **Describe this release** - Make sure to write what features this release includes, and any notable bugfixes. Also, thank all the folks who contributed to the release. You can find that information in a link that looks like this: `https://github.com/gomods/athens/compare/$PREVIOUS_TAG...$CURRENT_TAG`. Substitute `$PREVIOUS_TAG` for the last semver and `$CURRENT_TAG` to this one.
 
 When you're done, press the "Publish Release" button. After you do, our [Drone](https://cloud.drone.io) job will do almost everything.
 
-Make sure the Drone job finished, and check in Docker Hub to make sure the new release showed up in the [tags](https://hub.docker.com/r/gomods/athens/tags) section.
+Make sure the Drone CI/CD job finished, and check in Docker Hub to make sure the new release showed up in the [tags](https://hub.docker.com/r/gomods/athens/tags) section.
 
 ## Finishing up
 
 The Drone job will do everything except:
 
 - Tweet out about the new release
-- Update the helm chart
+- Update the helm chart in the `master` branch
```

Comment:
I wonder if this should be done as part of a pre-release commit, that way it also ends up on the release branch

Reply:
@twexler works for me. so it would be 

- update helm chart in master
- cut release branch 
- release etc...

is that what you had in mind?

Reply:
Yeah, although my one concern there is that without the latest docket image, the helm chart build may fail.

Reply:
Same here. That's why I'd prefer to make the releaser to wait until the new docker image is up before they manually submit the PR. Is that still cool with you? Let me know what you think

Reply:
Yeah, that’s fine.